tests/posts_tests/test_search_views.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
```

Comment:
```suggestion
from pytest_django.asserts import assertTemplateUsed
```

use `assertTemplateUsed`  https://docs.djangoproject.com/en/4.0/topics/testing/tools/#django.test.SimpleTestCase.assertTemplateUsed

Reply:
Why to use `assertTemplateUsed` instead of `response.templates[0].name`? they are both the same

Reply:
I think is more correct, and it is something built-in in Django so in my opinion it should be used.

Reply:
I replaced it with assertTemplateUsed